AUTOMATION_ENGINE_PLAN.md

```
@@ -0,0 +1,398 @@
+| Set Cost Center | Assign cost center |
+| Update Description | Modify/append description |
+| Create Transaction | Create new transaction (for webhooks) |
+| Send Push Notification | In-app + browser push |
+| Send Email | Via Resend |
+| Stop Execution | Halt rule processing |
+
+---
+
+## Package Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)